_data/glossary/approver.yaml

```diff
@@ -0,0 +1,10 @@
+id: approver
+name: Approver
+tags:
+- community
+short-description: |
+  Able to both review and approve K8s code contributions.
+long-description: |
+  While code review is focused on code quality and correctness, approval is focused on holistic acceptance of a contribution including&#58; backwards/forwards compatibility, adhering to API and flag conventions, subtle performance and correctness issues, interactions with other parts of the system, etc.
```

Comment:
While code review focuses on code quality and correctness, approval is focused on holistic acceptance of a contribution. Holistic acceptance includes backwards/forwards compatibility, adhering to API and flag conventions, subtle performance and correctness issues, interactions with other parts of the system, and others.